stdlib/CMakeLists.txt

```diff
@@ -38,6 +38,14 @@ else()
   set(SWIFT_STDLIB_STABLE_ABI_default FALSE)
 endif()
 
+if("${SWIFT_BUILD_SDK_OVERLAY}")
+  set(SWIFT_BUILD_TEST_SUPPORT_MODULES_default TRUE)
+elseif("${SWIFT_INCLUDE_TESTS}")
```

Comment:
The style in the project is to not quote the variable for these types of conditions:

```
if(SWIFT_BUILD_SDK_OVERLAY)
 ...
elseif(SWIFT_INCLUDE_TESTS)
 ...
else()
 ...
endif()
```

Is there a reason to not just do:

```
if(SWIFT_BUILD_SDK_OVERLAY OR SWIFT_INCLUDE_TESTS)
  ...
else()
 ...
endif()
```

Reply:
No reason. Let me fix that.